index.bs

```diff
@@ -326,9 +326,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
         Only the [=domain=] format of [=host=] is allowed here. This is for simplification and also is
         in recognition of various issues with using direct IP address identification in concert with
         PKI-based security.
-1. If the <code>|options|.{{CredentialRequestOptions/signal}}</code> is [=present=] and its
-    [=AbortSignal/aborted flag=] is set to [TRUE], return a {{DOMException}} whose name is "{{AbortError}}"
-    and terminate this algorithm.
+1. If the <code>|options|.{{CredentialRequestOptions/signal}}</code> is [=present=] and
```

Comment:
Can you update other places that return an exception instead of throwing in this algorithm, like L319 and L321-322 as well?

Reply:
Done :)